lomond/frame_parser.py

```diff
@@ -35,7 +35,9 @@ def __init__(self, frame_class=Frame, parse_headers=True):
     def parse(self):
         # Get the HTTP headers
         if self._parse_headers:
-            header_data = yield self.read_until(b'\r\n\r\n')
+            header_data = yield self.read_until(
```

Comment:
Adds a maximum header size. A sensible precaution, but doubt it will trip in practice.